readme

```diff
@@ -0,0 +1,18 @@
+hogehoge
+
+
+hogehogheo
+
+
+
+fl;aflksajf;lkjas
+
+
+l
+aaf;fkja;kfja;lkj;afklafjasf;a
```

Comment:
<table data-meta="generated_by_danger">
 <tbody>
   <tr>
     <td>:warning:</td>
     <td width="100%" data-sticky="false"><span data-href="https://github.com/mixi-minimo/minimo-client-android/blob/e80a3d67cae1752c0ecb962da317f4b6af485a80/minimoApp/src/main/java/jp/mixi/minimo/ui/reservation/SendReservationFragment.java#L337"></span>The result of <code>subscribe</code> is not used</td>
   </tr>
 </tbody>
</table>